src/BrightScriptDebugSession.ts

```diff
@@ -248,7 +279,7 @@ export class BrightScriptDebugSession extends DebugSession {
                 }
 
                 this.sendEvent(new CompileFailureEvent(compileErrors));
-                //TODO - shot gracefull
+                //TODO - shot graceful
```

Comment:
Not that this is a big deal, but i think the previous text was more grammatically correct. lol How about we just change it to 

```//stop the roku adapter and exit the channel```

Reply:
Done